Automation/Mobile/Appium/Locator naming differences.md

```diff
@@ -0,0 +1,87 @@
+
+To help with understanding, it is useful to use Appium Inspector to visualize the differences.
+
+Appium Inspector displays two useful sections on the right-hand side, Find By and Attribute.
+- **Find By** shows (some) locator strategies you could use to find the selected element, and the value it returns
+- **Attribute** provides information on the element's properties, and their corresponding values
+
+![inspecting_android_app](/img/locators_android.png)
+
+
+### AppiumBy.ID
+
+When locating an element using **AppiumBy.ID**, Appium looks for the `Resource ID` value on Android and the `accessibilityIdentifier` on iOS.
+
+
+### AppiumBy.ACCESSIBILITY_ID
+
+When locating an element using **AppiumBy.ACCESSIBILITY_ID**, Appium looks for `ContentDescription` value on Android, and both `accessibilityLabel` and `accessibilityIdentifier` on iOS. 
+
+Using `AppiumBy.ACCESSIBILITY_ID` may seem confusing, especially for iOS. Just think of it as using visible text to get an element.
+
+
+### Appium Inspector
+
+When inspecting the app with Appium Inspector you see the available strategies shown in the **Find By** section, and element attributes in the **Attribute** section.
+
+**Find By**
+
+For Android
+
+- `id`shows the value of `Resource ID`
+- `accessibility id` shows the value of `ContentDescription`
+
+For iOS
+
+- `accessibility id` shows the value of `accessibilityLabel` or `accessibilityIdentifier`
+
+
+**Attribute**
+
+For Android
+
+- `resource-id` shows the value of `Resource ID`
+- `content-desc` shows the value of `ContentDescription`
+
+For iOS
+
+- `name` shows the value of `accessibilityIdentifier`
+- `label` shows the value of `accessibilityLabel`
+
+
+### Table: Locator strategies in Appium Inspector
+
+The table below shows different locator strategies used in Appium, and corresponding strategy shown in Appium Inspector, and properties that are set on mobile platforms.
+
+| Appium Inspector   | Android            | iOS                                          |
+|:-------------------|:-------------------|:---------------------------------------------|
+| id                 | Resource ID        |                                              |
+| accessibility id   | ContentDescription | accessibilityLabel / accessibilityIdentifier |
+
+
+### Table: Element attributes in Appium Inspector
+
+The table below shows different element attributes found by Appium, and corresponding values shown in Appium Inspector, as well as properties that are set on mobile platforms.
+
+| Appium Inspector   | Android            | iOS                       |
+|:-------------------|:-------------------|:--------------------------|
+| resource-id        | Resource ID        |                           |
+| content-desc       | ContentDescription |                           |
+| name               |                    | accessibilityIdentifier   |
```

Comment:
Hey @dino-vinkovic 
Everything looks good from my perspective. 
Please just take a look at Matija's comment about "name attribute because accessibilityLabel is missing in 2 places".
When that is resolved, we can merge it to master.